mindsdb/integrations/utilities/handler_utils.py

```diff
@@ -0,0 +1,55 @@
+import os
+from typing import Dict
+
+from mindsdb.interfaces.storage.model_fs import HandlerStorage
+from mindsdb.utilities.config import Config
+
+"""Contains utilities to be used by handlers."""
+
+
+def get_api_key(
```

Comment:
Yes! This was sorely needed. @ZoranPandovski I'll open up a community issue once this is merged so that all other existing ML handlers that have copy-pasted this bit of code will now use the util instead 👍